content/pages/coc.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Pyninsula is dedicated to providing a harassment-free conference experience
+for everyone, regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race,
+age or religion.
+We do not tolerate harassment of conference participants in any form.
+Sexual language and imagery is not appropriate in the meetup,
+including talks.
+Pyninsula participants violating these rules may be sanctioned or
+expelled from the venue at the discretion of the meetup organizers.
+
+Harassment includes, but is not limited to:
+
+* Offensive verbal comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+Enforcement
+~~~~~~~~~~~
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately.
+
+If a participant engages in harassing behaviour,
+event organisers retain the right to take any actions to keep the event a
+welcoming environment for all participants.
+This includes warning the offender or expulsion from the meetup.
+
+Meetup organisers may take action to redress anything designed to,
+or with the clear impact of,
+disrupting the event or making the environment hostile for any participants.
+
+We expect participants to follow these rules at all meetup venues and
+meetup-related social activities.
+We think people should follow these rules outside event activities too!
+
+Reporting
+~~~~~~~~~
+
+If someone makes you or anyone else feel unsafe or unwelcome,
+please report it as soon as possible.
+Harassment and other code of conduct violations reduce the value of our event
+for everyone.
+We want you to be happy at our event.
+People like you make our event a better place.
+
+You can make a report either personally or anonymously.
+
+Anonymous Report
+~~~~~~~~~~~~~~~~
+
+You can make an `anonymous report`_.
+
+We can't follow up an anonymous report with you directly,
+but we will fully investigate it and take whatever action is necessary
+to prevent a recurrence.
+
+.. _anonymous report: https://goo.gl/forms/aDUSUvM8epVSip1z1
+
+Personal Report
```

Comment:
this might sound silly, but what do you think about practicing taking a personal report?  i think i might be surprised enough when it happen that i'll make dumb mistakes and missteps that'll make people more uncomfortable.

is a playbook too much?